src/components/TransactionHistory/TransactionHistoryList.tsx

```diff
@@ -1,58 +1,46 @@
-import { Center, Skeleton } from '@chakra-ui/react'
-import { memo } from 'react'
-import InfiniteScroll from 'react-infinite-scroll-component'
+import { ChevronDownIcon } from '@chakra-ui/icons'
+import { Button } from '@chakra-ui/react'
+import { memo, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
 import { CircularProgress } from 'components/CircularProgress/CircularProgress'
-import { Text } from 'components/Text'
 import { TransactionsGroupByDate } from 'components/TransactionHistory/TransactionsGroupByDate'
 import { useInfiniteScroll } from 'hooks/useInfiniteScroll/useInfiniteScroll'
 import { selectIsAnyTxHistoryApiQueryPending } from 'state/slices/selectors'
-import type { TxId } from 'state/slices/txHistorySlice/txHistorySlice'
+import { type TxId } from 'state/slices/txHistorySlice/txHistorySlice'
 import { useAppSelector } from 'state/store'
 
 type TransactionHistoryListProps = {
   txIds: TxId[]
   useCompactMode?: boolean
 }
 
-const scrollerStyle = { overflow: 'hidden' }
-
-const loader = (
-  <Center key={0}>
-    <CircularProgress isIndeterminate />
-  </Center>
-)
-
 export const TransactionHistoryList: React.FC<TransactionHistoryListProps> = memo(
   ({ txIds, useCompactMode = false }) => {
     const { next, data, hasMore } = useInfiniteScroll(txIds)
     const isAnyTxHistoryApiQueryPending = useAppSelector(selectIsAnyTxHistoryApiQueryPending)
+    const translate = useTranslate()
 
-    if (isAnyTxHistoryApiQueryPending && !data.length) {
-      return (
-        <>
-          {new Array(2).fill(null).map((_, i) => (
-            <Skeleton key={i} height={16} />
-          ))}
-        </>
-      )
-    }
+    const loadMoreRightIcon = useMemo(
+      () =>
+        isAnyTxHistoryApiQueryPending ? (
+          <CircularProgress isIndeterminate size={6} />
+        ) : (
+          <ChevronDownIcon />
+        ),
+      [isAnyTxHistoryApiQueryPending],
+    )
```

Comment:
This one look odd 
<img width="766" alt="image" src="https://github.com/shapeshift/web/assets/17035424/b6a3f310-28c3-4201-8ad8-791cc58c9f05">

As a user, I should either be able to click load more, or see a spinner, but not both?

Reply:
Sweet, I've updated it to only show the spinner if you reach the end of the data and its still being fetched